test/controllers/legal/sessions_controller_test.rb

```diff
@@ -0,0 +1,22 @@
+require 'test_helper'
+
+class Legal::SessionsControllerTest < ActionController::TestCase
+  test "#GET new renders" do
+    get :new
+
+    assert_response :ok
+  end
+
+  test "#POST create signs in with valid user" do
+    post :create, session: { email: 'david.attenborough@bbc.co.uk', password: 'test' }
```

Comment:
Line is too long. [86/80]<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.